cmd/update.go

```diff
@@ -55,8 +56,14 @@ func newUpdateCmd() *updateCmd {
 
 			// Update single binary
 			if bin != "" {
+				path, errBin := exec.LookPath(bin)
```

Comment:
I think we can be a bit smarter here and avoid the lookup if the user already sends us the path right?  If I do `bin update $(which kind)` like I do today, seems like the lookup is unnecessary. WDYT?

Reply:
Yes, maybe i would add a regexp check for the value / in the path value before call the LookPath function.

But on the other hand i looked at the LookPath docs and i found the is a check of the filestring
"If file contains a slash, it is tried directly and the PATH is not consulted." 
https://golang.org/src/os/exec/lp_unix.go?s=923:965#L24

I'm not sure if this validation of the LookPath function would be enough WDYT?

Reply:
> I'm not sure if this validation of the LookPath function would be enough WDYT?

Yes, I saw that as well, but regardless if the argument contains a "/" or not, it still calls the `Stat` syscall afterwards to check if the file exists (https://go.googlesource.com/go/+/go1.15.8/src/os/exec/lp_unix.go#20). I guess this is premature optimization and it's not really necessary. Let's leave it like this for the moment, I don't think we need to change it